src/pyAMReX.cpp

```diff
@@ -43,7 +43,7 @@ void init_ParticleContainer(py::module &);
 void init_Periodicity(py::module &);
 void init_PODVector(py::module &);
 void init_Vector(py::module &);
-
+void init_Utility(py::module &);
```

Comment:
This could be alphabetical for clarity as well, went a bit astray in recent PRs :sweat_smile:

Reply:
yeah -- we were going to keep it alphabetical but saw that it was not really that way 

Then we discussed that may be we can keep a distinction between amrex classes and functions and left a space there

Reply:
what would be the best practice here, keeping all class wrappers together i.e., separate from functions wrappers -- or merge them , but keep it alphabetical

Reply:
For example, somewhere in that file, RealBox comes before Before Array 
not alphabetical

Reply:
Yes, there slipped in some other inconsistencies here that I will fix separately.

For docs and declarations, let's keep them alphabetical for easy reading.
On the call site, we need to be careful to call dependent classes and functions that use AMReX types after we declared them.